src/foam/nanos/theme/permissions.jrl

```diff
@@ -0,0 +1,2 @@
+p({"class":"foam.nanos.auth.Permission","id":"theme.rw.primary1","description":"Update primary1"})
+p({"class":"foam.nanos.auth.Permission","id":"theme.rw.primary2","description":"Update primary2"})
```

Comment:
Why do we add these two permissions manually? Shouldn't writePermissionsRequired add these?

Reply:
we don't have that kind of support.